### ISSUE TITLE
Any plan to implement DSFD for face detection

### ISSUE THREAD

**Issue by fl82hope** [open-mmlab/mmdetection#6720]
The provided  config  https://github.com/open-mmlab/mmdetection/blob/master/configs/wider_face/ssd300_wider_face.py, the training loss is always Nan, Any solutions?

DSFD (https://github.com/Tencent/FaceDetection-DSFD) is a practical face detector, any plan to incorporate it into mmdetection?

**Comment by AronLin** [comment]
For the second question, we mainly support general object detection, so we have not plan to reimplement it.
For the first question, how many GPUs did you use? You can set `grad_clip` in the `optimizer_config`.
